docs/_layouts/global.html

```diff
@@ -17,113 +19,133 @@
+                            <a class="dropdown-item" href="cluster-overview.html">Overview</a>
+                            <a class="dropdown-item" href="submitting-applications.html">Submitting Applications</a>
+                            <div class="dropdown-divider"></div>
+                            <a class="dropdown-item" href="spark-standalone.html">Spark Standalone</a>
+                            <a class="dropdown-item" href="running-on-mesos.html">Mesos</a>
+                            <a class="dropdown-item" href="running-on-yarn.html">YARN</a>
+                            <a class="dropdown-item" href="running-on-kubernetes.html">Kubernetes</a>
+                        </div>
+                    </li>
+
+                    <li class="nav-item dropdown">
+                        <a href="#" class="nav-link dropdown-toggle" id="navbarMore" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">More</a>
+                        <div class="dropdown-menu" aria-labelledby="navbarMore">
+                            <a class="dropdown-item" href="configuration.html">Configuration</a>
+                            <a class="dropdown-item" href="monitoring.html">Monitoring</a>
+                            <a class="dropdown-item" href="tuning.html">Tuning Guide</a>
+                            <a class="dropdown-item" href="job-scheduling.html">Job Scheduling</a>
+                            <a class="dropdown-item" href="security.html">Security</a>
+                            <a class="dropdown-item" href="hardware-provisioning.html">Hardware Provisioning</a>
+                            <a class="dropdown-item" href="migration-guide.html">Migration Guide</a>
+                            <div class="dropdown-divider"></div>
+                            <a class="dropdown-item" href="building-spark.html">Building Spark</a>
+                            <a class="dropdown-item" href="https://spark.apache.org/contributing.html">Contributing to Spark</a>
+                            <a class="dropdown-item" href="https://spark.apache.org/third-party-projects.html">Third Party Projects</a>
+                        </div>
+                    </li>
+
+                    <li class="nav-item">
+                        <input type="text" id="docsearch-input" placeholder="Search the docs…">
+                    </li>
+                </ul>
+                <!--<span class="navbar-text navbar-right"><span class="version-text">v{{site.SPARK_VERSION_SHORT}}</span></span>-->
+            </div>
+        </nav>
+
+        {% if page.url == "/" %}
+            <section class="hero-banner position-relative">
+            <div class="bg">
+            </div>
+            <div class="container position-relative">
+                <div class="row">
+                  <h1 style="max-width: 680px;">Apache Spark - A Unified engine for large-scale data analytics</h1>
```

Comment:
The Spark website is using `Unified engine for large-scale data analytic`. Shall we keep them consistent?
<img width="736" alt="image" src="https://user-images.githubusercontent.com/1097932/226060783-39e3f397-9ec7-41ec-831a-4ba61629a74c.png">

Reply:
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/1097932/226061110-3bf3dbfa-8777-40b2-94ce-81806c82bebe.png">

Reply:
note: I don't have a strong opinion on this one.

Reply:
The reason I added Apache Spark explicitly is because on the main website there is a banner on top saying explicit Apache Spark whereas here we don't have the top banner.